docs/pom.xml

```diff
@@ -37,18 +37,15 @@
                 <artifactId>asciidoctor-maven-plugin</artifactId>
             </plugin>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-assembly-plugin</artifactId>
+            </plugin>
+            <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>build-helper-maven-plugin</artifactId>
             </plugin>
         </plugins>
     </build>
 
-    <repositories>
```

Comment:
I don't think so. I thought it was something I'd added by mistake. Back in it goes.